EIPS/eip-7642.md

```diff
@@ -1,7 +1,7 @@
 ---
 eip: 7642
-title: eth/69 - Drop pre-merge fields
-description: Drop unnecessary fields after the merge
+title: eth/69 - history expiry and simpler receipts
+description: Adds history serving window and removes bloom filter in receipt
 author: Marius van der Wijden (@MariusVanDerWijden), Felix Lange <fjl@ethereum.org>
```

Comment:
```suggestion
author: Marius van der Wijden (@MariusVanDerWijden), Felix Lange (@fjl)
```
this allows you to merge changes to the eip via the bot

Reply:
Yeah I know but I want to discuss a bit before merging.